{"name": "test_six[6-9-13]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 15 == 13\n +  where 15 = add(6, 9)", "trace": "self = <Work.test_work.TestB object at 0x10b4942b0>, a = 6, b = 9, expected = 13\n\n    @pytest.mark.parametrize(\"a,b,expected\", json.load(open(\"calc.json\")))\n    def test_six(self, a, b, expected):\n        logging.info(\"This is test_six!!!\")\n>       assert add(a, b) == expected\nE       assert 15 == 13\nE        +  where 15 = add(6, 9)\n\ntest_work.py:58: AssertionError"}, "attachments": [{"name": "log", "source": "7f2be5d5-60fa-4307-a6b8-a03611abc229-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "e8856c91-f1d3-4e05-98f5-4128b34e20ee-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "a", "value": "6"}, {"name": "b", "value": "9"}, {"name": "expected", "value": "13"}], "start": 1570674115245, "stop": 1570674115246, "uuid": "64b23f57-1b84-4f91-83b8-d655bb1dd08d", "historyId": "fe82b554876d75cd5a307d34c53dd79f", "testCaseId": "33cda52eda96a5e8528fcc03f1cd347a", "fullName": "test_work.TestB#test_six", "labels": [{"name": "suite", "value": "test_work"}, {"name": "subSuite", "value": "TestB"}, {"name": "host", "value": "ParsondeMacBook-Pro.local"}, {"name": "thread", "value": "27998-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_work"}]}